packages/api-explorer/src/components/SelectorContainer/SdkLanguageSelector.spec.tsx

```diff
@@ -82,4 +97,20 @@ describe('SdkLanguageSelector', () => {
       })
     })
   })
+  test('changing SDK language causes parameter to be pushed to URL', async () => {
+    renderWithRouterAndReduxProvider(<SdkLanguageSelector />)
```

Comment:
discuss with Joseph: whitespace between the 3 main steps, render, getting components, interacting with components.

Be generous with it between different tests.